core/templates/dev/head/App.ts

```diff
@@ -75,6 +75,12 @@ require('google-analytics.initializer.ts');
 // loaded after app.constants.ts
 require('I18nFooter.ts');
 
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import {WindowDimensionsService} from
```

Comment:
Add space after `{` and before `}`

Reply:
Oh we are using ```UpgradedServices``` now so i don't need to import this anymore in ```App.ts```, hence remove that. Thanks, good catch!

Reply:
Done

Reply:
IMO, we will need import here.

Reply:
Please remove the require statement for window dimension service in App.ts. Its no longer needed.

Reply:
@bansalnitish Yup, did that.